File: C#/URI/1049.cs
using System; 

class URI {

    static void Main(string[] args) { 
        String  a, b, c;

        a = Convert.ToString(Console.ReadLine());
        b = Convert.ToString(Console.ReadLine());
        c = Convert.ToString(Console.ReadLine());
        sigla = Convert.ToString(Console.ReadLine());


        if (String.Compare(a, "vertebrado")==0 && String.Compare(b, "ave") == 0 && String.Compare(c, "carnivoro") == 0)
        {
            Console.WriteLine("aguia");
        }
        if (String.Compare(a, "vertebrado") == 0 && String.Compare(b, "ave") == 0 && String.Compare(c, "onivoro") == 0)
        {
            Console.WriteLine("pomba");
        }
        if (String.Compare(a, "vertebrado") == 0 && String.Compare(b, "mamifero") == 0 && String.Compare(c, "onivoro") == 0)
        {
            Console.WriteLine("homem");
        }
        if (String.Compare(a, "vertebrado") == 0 && String.Compare(b, "mamifero") == 0 && String.Compare(c, "herbivoro") == 0)
        {
            Console.WriteLine("vaca");
        }
        if (String.Compare(a, "invertebrado") == 0 && String.Compare(b, "inseto") == 0 && String.Compare(c, "hematofago") == 0)
        {
            Console.WriteLine("pulga");
        }
        if (String.Compare(a, "invertebrado") == 0 && String.Compare(b, "inseto") == 0 && String.Compare(c, "herbivoro") == 0)
        {
            Console.WriteLine("lagarta");
        }
        if (String.Compare(a, "invertebrado") == 0 && String.Compare(b, "anelideo") == 0 && String.Compare(c, "hematofago") == 0)
        {
            Console.WriteLine("sanguessuga");
        }
        if (String.Compare(a, "invertebrado") == 0 && String.Compare(b, "anelideo") == 0 && String.Compare(c, "onivoro") == 0)
        {
            Console.WriteLine("minhoca");
        }
    }

}

File: JS/URI/1049.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var a =lines.shift().trim()
var b =lines.shift().trim()
var c =lines.shift().trim()


if(a == "vertebrado" && b == "ave" && c == "carnivoro")
{
    console.log("aguia\n");
}
if(a == "vertebrado" && b == "ave" && c == "onivoro")
{
    console.log("pomba\n");
}
if(a == "vertebrado" && b == "mamifero" && c == "onivoro")
{
    console.log("homem\n");
}
if(a == "vertebrado" && b == "mamifero" && c == "herbivoro")
{
    console.log("vaca\n");
}
if(a == "invertebrado" && b == "inseto" && c == "hematofago")
{
    console.log("pulga\n");
}
if(a == "invertebrado" && b == "inseto" && c == "herbivoro")
{
    console.log("lagarta\n");
}
if(a == "invertebrado" && b == "anelideo" && c == "hematofago")
{
    console.log("sanguessuga\n");
}
if(a == "invertebrado" && b == "anelideo" && c == "onivoro")
{
    console.log("minhoca\n");
}